tests/unit_test/callback/pyscript_handler_test.py

```
@@ -1224,6 +1370,106 @@ def test_fetch_collection_data_empty_result():
+            "pincode": 538494
+        }
+    }
+
+    mock_document = MagicMock()
+    mock_document.to_mongo.return_value.to_dict.return_value = mock_data
+
+    mock_queryset = MagicMock()
+    mock_queryset.first.return_value = mock_document
+
+    with pytest.raises(Exception, match="Missing bot id"):
+        result = PyscriptSharedUtility.get_crud_metadata('testing_crud_api', 'test_user')
```

Ruff (0.11.9)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)